pages/tutorials/callbacks.mdx

```diff
@@ -1,179 +0,0 @@
-import { Code } from "../../components/Code/Code";
```

Comment:
We need to write a new guide about callbacks, which is more use-case-driven ( _i.e in which case callbacks are uself_ ) rather than being a copy of the API reference 💭